concepts/60 Themes and Styles/05 Predefined Themes/50 Apply a Theme.md

```diff
@@ -12,14 +12,30 @@ Configure the module bundler you are using and import `dx.common.css` and a them
 2. Open the DevExtremeBundleConfig.cs file
 3. Find the following code and replace `dx.light.css` in it with the theme stylesheet of your choice:
 
+            <!-- tab: DevExtremeBundleConfig.cs -->
             styleBundle
                 .Include("~/Content/dx.common.css")
                 .Include("~/Content/dx.light.css");
 
+4. Open the Views/Shared/_Layout.cshtml file and add the `dx-viewport` class to the `<body>` tag. This ensures that theme colors and typography settings are applied to all elements on the page, not only to DevExtreme widgets.
```

Comment:
Can we change the second sentence to:
This ensures that theme colors and typography settings are applied to all page elements (and not only to DevExtreme widgets).